src/components/TodoItem.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from "react";
+
+class TodoItem extends Component {
+  render() {
+    return (
+      <li>
+        <input
+          type="checkbox"
+          checked={this.props.todo.completed}
+          onChange={() => {
+            this.props.handleChangeProps(this.props.todo.id);
+          }}
+        />
+        <button onClick={() => this.props.deleteTodoProps(this.props.todo.id)}>
+          Delete
+        </button>
+        {this.props.todo.title}
```

Comment:
good job, but I will recond you to use object destructing to get (id, title, completed) 
so you do not need to write `this.props.todo.`  it takes time and its not good for maintaining the code
it would be better to use this inside `render()`  before the `return` like this:-
``` const {id, title, compelted} = this.props.todo; ```
then you can use `id` instead `this.props.todo.id`